src/test/javascript/integration/time_next_seconds_spec.js

```diff
@@ -19,46 +32,27 @@ let currentTime = null;
 describe(__filename, function () {
     beforeAll(async () => {
         const response = await createChannel(channelName);
-        expect(getProp('statusCode', response)).toBe(201);
+        expect(getProp('statusCode', response)).toEqual(201);
     });
 
-    it('gets times', function (done) {
-        const url = channelResource + '/time';
-        request.get({url: url, json: true},
-            function (err, response, body) {
-                expect(err).toBeNull();
-                expect(response.statusCode).toBe(200);
-                stableTime = moment(body.stable.millis).utc();
-                currentTime = moment(body.now.millis).utc();
-                done();
-            });
+    it('gets times', async () => {
+        const response = await hubClientGet(`${channelResource}/time`, headers);
+        expect(getProp('statusCode', response)).toEqual(200);
+        const stableTimeMillis = fromObjectPath(['body', 'stable', 'millis'], response);
+        const currentTimeMillis = fromObjectPath(['body', 'stable', 'millis'], response);
```

Comment:
this should probably be `now` instead of `stable`.

Reply:
derp